src/qml/pages/node/Peers.qml

```diff
@@ -31,36 +30,89 @@ Page {
         horizontalAlignment: Text.AlignHCenter
     }
 
+    RowLayout {
```

Comment:
Maybe make this [Flow](https://doc.qt.io/qt-6/qml-qtquick-flow.html) so that the buttons don't go out of view on narrow screens?

| PR | Patch |
| -- | ----- |
| ![2023-02-21_15-05](https://user-images.githubusercontent.com/23396902/220451122-e0dcf0e4-60cc-4b90-aca2-9597c95dc4ae.png) | ![2023-02-21_15-29](https://user-images.githubusercontent.com/23396902/220451421-3e607277-9d5b-40bc-a8c1-b1ab5776118f.png) |

If we want the buttons to always be centered within the width of the flow, we can add some additional logic to the left and right margins




```suggestion
    Flow {
```

Reply:
Neat QML type/item.  I like it.